README.md

```diff
@@ -46,3 +46,24 @@ overlap between the query and document title:
 - "translate bytes to doubles" returns "Convert Bytes to Floating Point Numbers in Python"
 
 Note that in other cases, the results can be noisy and unintuitive. For example, "zipping up files" also assigns high scores to "Partial .csproj Files" and "How to avoid .pyc files?".
+
+## Install using Docker
+
+Run this docker command to install the required packages and start the application.
+
+docker run --name text_embeddings  -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node"  -d shantanuo/est
```

Comment:
I tried running this but I'm not actually able to find `shantanuo/est` on docker hub. It would also be nice to avoid linking to your personal repository (so you aren't on the hook for maintaining it :))

For now I think we can just combine these two sections under one `Build and Run from Docker`, and I can work on uploading an image if there's interest.